### DISCUSSION TITLE
saf-training/courses/beginner/03

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#276] [Ideas]
# saf-training/courses/beginner/03

Studying an InSpec profile The best way to learn how an InSpec profile works is to write our own example. Let's start out with a simple profile and start adding more tests to ma...

https://mitre.github.io/saf-training/courses/beginner/03.html

**Comment by karikarshivani** [comment]
Change the lab env `my_nginx` folder name to `my_nginx_answer_key` to allow the init command to run

**Comment by p-oneil** [comment]
Moved to PR
